File: src/main/java/com/mirrormirror/controller/ErrorHandler.java
package com.mirrormirror.controller;

import org.springframework.web.bind.annotation.ControllerAdvice;
import org.springframework.web.bind.annotation.ExceptionHandler;

@ControllerAdvice
public class ErrorHandler {
    @ExceptionHandler(RuntimeException.class)
    public void handleException(RuntimeException e){
        System.out.println(e.getMessage());
    }
}

File: src/main/java/com/mirrormirror/controller/ErrorHandler.kt
package com.mirrormirror.controller

import org.springframework.web.bind.annotation.ControllerAdvice
import org.springframework.web.bind.annotation.ExceptionHandler

@ControllerAdvice
class ErrorHandler {
    @ExceptionHandler(RuntimeException::class)
    fun handleException(e: RuntimeException) {
        println(e.message)
    }
}
